frontend/src/Components/PostComponent.module.scss

```diff
@@ -6,6 +6,10 @@
   margin-bottom: 32px;
 }
 
+:global(.internalLinkExpand) .post {
+  margin-bottom: 12px;
```

Comment:
Let's add a border to separate expand content from the rest of the post/comment - currently it is hard to tell which is which

Reply:
But there is a border already (on the left side).

Reply:
Imho it is very subtle and hardly visible - it mixes up with comment layers.

Reply:
Made the border brighter:
<img width="430" alt="image" src="https://github.com/spaceshelter/orbitar/assets/2865203/e156199a-150c-47ae-9188-e3157806760d">